webapp/metviewer/js/metviewer_common.js

```diff
@@ -227,6 +227,7 @@ value_to_desc_map['ORANK_CLIMO'] = 'The value of the inluded climatology';
 value_to_desc_map['ORANK_ENS_MEAN_OERR'] = 'The PERTURBED ensemble mean';
 value_to_desc_map['ORANK_SPREAD_OERR'] = 'The spread of the PERTURBED ensemble member values';
 value_to_desc_map['ORANK_SPREAD_PLUS_OERR'] = 'The square root of the sum of the unperturbed ensemble variance and the observation error variance';
+value_to_desc_map['RPSS_COMP'] = 'Complement of the Ranked Probability Score';
```

Comment:
There is a typo on this line.  "RPSS_COMP" should really be "RPS_COMP".  Since the change was already merged into develop, GitHub doesn't give me an option of proposing a change.  Nor does it allow me to "Complete my review".